### PR TITLE
zh: /blog/2019/cve-2019-12243/index.md

### PR DESCRIPTION
**Pull request by fleeto**

### CODE REVIEW

**Comment by fleeto** [comment]
`[trans-bot:pushed]`

Thank you @fleeto, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/1011) had been updated.

File: content_zh/blog/2019/cve-2019-12243/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: 安全更新 - CVE-2019-12243
+description: 应对 CVE-2019-12243 带来的安全威胁。
+publishdate: 2019-05-28
+attribution: The Istio Team
+---
+
+在对 [Istio 1.1.7](/zh/about/notes/1.1.7) 的发行声明进行评审时，我们意识到这一版本中修复的 [Issue 13868](https://github.com/istio/istio/issues/13868)，实际上是一个安全漏洞。
+
+起初我们认为这个 Bug 影响的是 [TCP 认证](/zh/about/feature-stages/#安全和策略实施)，这一功能仍然处于 Alpha 阶段，因此无需启动安全顾问流程，然而接下来我们意识到 [Deny Checker](/docs/reference/config/policy-and-telemetry/adapters/denier/) 和 [List Checker](/docs/reference/config/policy-and-telemetry/adapters/list/) 这两个稳定版功能也受到了影响。我们对流程进行了复查，确认将应该进行[披露](/about/security-vulnerabilities/)的安全漏洞当成了普通 Bug。
```
Comment (SataQiu) [inline content_zh/blog/2019/cve-2019-12243/index.md]:
```suggestion
起初我们认为这个 Bug 影响的是 [TCP 认证](/zh/about/feature-stages/#安全和策略实施)，这一功能仍然处于 Alpha 阶段，因此无需启动安全顾问流程，然而接下来我们意识到 [Deny Checker](/zh/docs/reference/config/policy-and-telemetry/adapters/denier/) 和 [List Checker](/zh/docs/reference/config/policy-and-telemetry/adapters/list/) 这两个稳定版功能也受到了影响。我们对流程进行了复查，确认将应该进行[披露](/zh/about/security-vulnerabilities/)的安全漏洞当成了普通 Bug。
```

Comment (fleeto) [reply content_zh/blog/2019/cve-2019-12243/index.md]: 我说过了 Reference 部分不再引用中文版

Comment (SataQiu) [reply content_zh/blog/2019/cve-2019-12243/index.md]: OK!

**Pull request merged**